diff_cover/violations_reporter.py

```diff
@@ -116,14 +116,14 @@ def _get_classes(self, xml_document, src_path):
              src_abs_path in [
                  os.path.join(
                      source,
-                     clazz.get('filename')
+                     os.path.normpath(clazz.get('filename').replace("\\", "/"))
                  ) for source in sources]]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_abs_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_abs_path]
             or
             [clazz for clazz in classes if
-             clazz.get('filename') == src_rel_path]
+             os.path.normpath(clazz.get('filename').replace("\\", "/")) == src_rel_path]
```

Comment:
You actually caught me being a bit sloppy here. I meant to delete the `os.path.normpath`. First I tried `os.path.normpath` but nothing changed. I think because I was writing this on a mac. Since, as you noticed, git seems to be using unix style paths anyway I think I just want to convert the ones that come in from the xml.

Tonight when i get home ill remove the calls to normpath here. I *think* that will have the desired effect.